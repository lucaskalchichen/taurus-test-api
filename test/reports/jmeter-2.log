2025-05-18 21:57:28,505 INFO o.a.j.u.JMeterUtils: Setting Locale to en_US
2025-05-18 21:57:28,516 INFO o.a.j.JMeter: Loading user properties from: /root/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-18 21:57:28,516 INFO o.a.j.JMeter: Loading system properties from: /root/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-18 21:57:28,516 INFO o.a.j.JMeter: Loading additional properties from: /tmp/artifacts/jmeter-bzt-2.properties
2025-05-18 21:57:28,517 INFO o.a.j.JMeter: Setting System properties from file: /tmp/artifacts/system-2.properties
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: Version 5.5
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: os.name=Linux
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: os.arch=amd64
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-18 21:57:28,521 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-18 21:57:28,522 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-18 21:57:28,522 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-18 21:57:28,522 INFO o.a.j.JMeter: Available Processors =28
2025-05-18 21:57:28,525 INFO o.a.j.JMeter: Default Locale=English (United States)
2025-05-18 21:57:28,525 INFO o.a.j.JMeter: JMeter  Locale=English (United States)
2025-05-18 21:57:28,525 INFO o.a.j.JMeter: JMeterHome=/root/.bzt/jmeter-taurus/5.5
2025-05-18 21:57:28,525 INFO o.a.j.JMeter: user.dir  =/bzt-configs
2025-05-18 21:57:28,525 INFO o.a.j.JMeter: PWD       =/bzt-configs
2025-05-18 21:57:28,526 INFO o.a.j.JMeter: IP: 172.18.0.4 Name: 96e8f23a7950 FullName: 96e8f23a7950
2025-05-18 21:57:28,526 INFO o.a.j.JMeter: user.classpath=/tmp/artifacts:/tmp/artifacts
2025-05-18 21:57:28,526 INFO o.a.j.JMeter: Adding to classpath and loader: /tmp/artifacts
2025-05-18 21:57:28,527 INFO o.a.j.JMeter: Adding to classpath and loader: /tmp/artifacts
2025-05-18 21:57:28,532 INFO o.a.j.s.FileServer: Default base='/bzt-configs'
2025-05-18 21:57:28,533 INFO o.a.j.s.FileServer: Set new base='/tmp/artifacts'
2025-05-18 21:57:28,613 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-18 21:57:28,623 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-18 21:57:28,625 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-18 21:57:28,627 INFO o.a.j.s.SaveService: Loading file: /tmp/artifacts/modified_requests-2.jmx
2025-05-18 21:57:28,646 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-18 21:57:28,647 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-18 21:57:28,654 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:57:28,654 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:57:28,654 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:57:28,654 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:57:28,654 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-18 21:57:28,655 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-18 21:57:28,669 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-18 21:57:28,670 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-18 21:57:28,670 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-18 21:57:28,676 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-18 21:57:28,676 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-18 21:57:28,821 INFO o.a.j.JMeter: Running test (1747605448820)
2025-05-18 21:57:28,831 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-18 21:57:28,831 INFO o.a.j.e.StandardJMeterEngine: Starting 1 threads for group api-test.
2025-05-18 21:57:28,831 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-18 21:57:28,833 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-18 21:57:28,833 INFO c.b.j.t.c.ConcurrencyThreadStarter: Start supplying threads
2025-05-18 21:57:30,355 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-18 21:57:30,355 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-18 21:57:30,355 INFO o.a.j.t.JMeterThread: Thread started: api-test-ThreadStarter 1-1
2025-05-18 21:57:30,356 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-18 21:57:30,356 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-18 21:57:30,370 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = 96e8f23a7950
2025-05-18 21:57:30,373 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-18 21:57:42,331 INFO c.b.j.t.c.ConcurrencyThreadStarter: Done supplying threads
2025-05-18 21:57:42,489 INFO o.a.j.t.JMeterThread: Thread is done: api-test-ThreadStarter 1-1
2025-05-18 21:57:42,490 INFO o.a.j.t.JMeterThread: Thread finished: api-test-ThreadStarter 1-1
2025-05-18 21:57:42,491 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-18 21:57:42,495 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
