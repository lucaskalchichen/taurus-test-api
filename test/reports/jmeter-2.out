Created the tree successfully using /tmp/artifacts/modified_requests-2.jmx
Starting standalone test @ 2025 May 18 21:57:28 UTC (1747605448666)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 18 21:57:42 UTC (1747605462494)
... end of run
